### PR TITLE
Show current DataBreakpoint access type in "Breakpoints" view (#119588)

### PR DESCRIPTION
**Pull request by yannickowow**
This PR fixes #https://github.com/microsoft/vscode/issues/119588

image

Not sure if I do it correctly here:
```ts
if (dataBreakpoint.accessType) {
	const accessType = dataBreakpoint.accessType === 'read' ? localize('read', "Read") : dataBreakpoint.accessType === 'write' ? localize('write', "Write") : localize('access', "Access");
	data.accessType.textContent = accessType;
}
```

### CODE REVIEW

File: src/vs/workbench/contrib/debug/browser/breakpointsView.ts [hunk 1]
```diff
@@ -622,6 +627,10 @@ class DataBreakpointsRenderer implements IListRenderer<DataBreakpoint, IBaseBrea
 		if (session && !session.capabilities.supportsDataBreakpoints) {
 			data.breakpoint.title = localize('dataBreakpointsNotSupported', "Data breakpoints are not supported by this debug type");
 		}
+		if (dataBreakpoint.accessType) {
+			const accessType = dataBreakpoint.accessType === 'read' ? localize('read', "Read") : dataBreakpoint.accessType === 'write' ? localize('write', "Write") : localize('access', "Access");
+			data.accessType.textContent = accessType;
+		}
```
Comment (isidorn) [inline src/vs/workbench/contrib/debug/browser/breakpointsView.ts:633]: We need the `else` case here
`data.accessType.textContent = ''`
Reason is that these templates are reused between different elements, so we need to clear out the field if the data breakpoint does not have an `accessType`

**Review by isidorn — CHANGES_REQUESTED** [review]

**Comment by isidorn** [comment]
@yannickowow looks great, thanks a lot for this PR!
I have only found one issue and I have commented in the code. Once you tackle that I will happily merge this in.

File: src/vs/workbench/contrib/debug/browser/breakpointsView.ts (hunk 1 above)
Comment (yannickowow) [reply src/vs/workbench/contrib/debug/browser/breakpointsView.ts:635]: Done in last commit

**Comment by yannickowow** [comment]
It should be okay now!

**Comment by isidorn** [comment]
@yannickowow thanks a lot, this looks great, merging in ☀️ 👏

**Pull request merged**